dmreactplugin/src/components/common/pinnedMessage/button.jsx

```diff
@@ -1,17 +1,21 @@
 import React from "react";
 import style from "styled-components";
 
```

Comment:
hey whats up please pull from upstream if this merge is added you are going to affect my component badly

Reply:
we are closing the pull request